src/main/java/core/basesyntax/service/ReadDailyTransactions.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.DataBase;
+import core.basesyntax.db.DataBaseCsvImpl;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReadDailyTransactions {

```

Comment:
```suggestion
public class FileReaderImpl implements FileReader {
```
you should have abstract types that define functionality and implementation